static/js/execution.js

```diff
@@ -77,47 +77,90 @@ export async function execute(command) {
 
     let out = [];
 
+    for (let table of dbRet) {
+        out.push(tableFormat(table));
+    }
+
     if (visualization) {
         visualization.push(tableFormat(dbRet[0]));
 
         let visualizeButton = document.createElement("BUTTON");
+        $(visualizeButton).addClass("btn btn-info btn-sm");
         visualizeButton.innerHTML = "Visualize";
         out.push(visualizeButton);
 
-        let visualizePane = $.parseHTML("<div class='visualize-pane'' </div>");
+        let visualizePane = document.createElement("DIV");
+        $(visualizePane).addClass('card');
+        let visualizePaneHeader = document.createElement("DIV");
+        $(visualizePaneHeader).addClass('card-header');
         let innerVisualizePane = document.createElement("DIV");
+        $(innerVisualizePane).addClass('card-body');
+
+        let firstButton = document.createElement("BUTTON");
+        firstButton.innerHTML = "&lt;&lt;";
+        $(firstButton).addClass("btn btn-secondary");
 
         let prevButton = document.createElement("BUTTON");
-        prevButton.innerHTML = "prev";
+        prevButton.innerHTML = "&lt;";
+        $(prevButton).addClass("btn btn-secondary");
 
         let nextButton = document.createElement("BUTTON");
-        nextButton.innerHTML = "next";
+        nextButton.innerHTML = "&gt;";
+        $(nextButton).addClass("btn btn-secondary");
+
+        let lastButton = document.createElement("BUTTON");
+        lastButton.innerHTML = "&gt;&gt;";
+        $(lastButton).addClass("btn btn-secondary");
+
+        let buttons = document.createElement("DIV");
+        $(buttons).addClass("btn-group visButtons");
```

Comment:
```suggestion
        $(buttons).addClass("btn-group  btn-group-sm visButtons");
```
Would this look better?